src/current/v23.1/sso-db-console.md

```diff
@@ -6,94 +6,105 @@ docs_area: manage
+
+This page describes:
+
+- The login flow for users, once a cluster is SSO-enabled.
+- Configuring a CockroachDB Cluster for OIDC SSO authentication to the DB Console.
+- Provioning an OIDC client for CockroachDB Cluster SSO authentication, using GCP.
+
+Additional configuration is needed to enable SSO authentication for SQL clients, which is described here: [Cluster Single Sign-on (SSO) using a JSON web token (JWT)]({% link {{ page.version.version }}/sso-sql.md %}) 
 
 {{site.data.alerts.callout_info}}
-SSO for DB Console is available to CockroachDB {{ site.data.products.cloud }} customers.
+SSO for DB Console is available to CockroachDB {{ site.data.products.dedicated }} customers.
 
 On CockroachDB {{ site.data.products.core }} clusters, SSO for DB Console requires an [Enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}).
 {{site.data.alerts.end}}
 
-## Requirements
+This SSO implementation uses the [authorization code grant type](https://tools.ietf.org/html/rfc6749#section-4.1) to authenticate a user.
 
-- An external OAuth 2.0 identity provider
+**Prerequisites**:
 
-## Login flow
+- **IdP:** You must have access to an OAuth 2.0 identity provider and client. The process of provisioning one with Google Cloud Platform is described here: [Provisioning an OAuth client using Google Cloud Platform (GCP)](#provision-an-oauth-client-using-google-cloud-platform-gcp)
+
+- **CockroachDB Cluster**: you must have access to one of the following:
+	1. A {{ site.data.products.core }} cluster enabled with a valid [CockroachDB Enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}).
+	1. A [CockroachDB {{ site.data.products.dedicated }}cluster](https://www.cockroachlabs.com/docs/cockroachcloud/create-your-cluster).
 
-This SSO implementation uses the [authorization code grant type](https://tools.ietf.org/html/rfc6749#section-4.1) to authenticate a user.
 
-1. A user opens the DB Console and clicks on the OAuth login button.
+## Log in to a cluster's DB Console with SSO
+
+From the user's perspective, once the cluster is properly configured to an identity provider, the sign-in flow is as follows:
+
+1. A user opens the cluster's DB Console, and clicks on the OAuth login button which renders.
 1. The user is redirected to an external identity provider.
 1. The user successfully authenticates with the provider, completing the OAuth flow.
-1. The user is redirected to the CockroachDB cluster via a callback URL.
-1. The authorization code in the callback query is exchanged for an [ID Token](https://openid.net/specs/openid-connect-core-1_0.html#IDToken).
-1. CockroachDB matches the ID Token to a registered SQL user.
-1. CockroachDB creates a web session for the SQL user.
-1. The user is redirected to the [DB Console Cluster Overview]({% link {{ page.version.version }}/ui-cluster-overview-page.md %}).
+1. The user is redirected to the CockroachDB cluster.
```

Comment:
Here, not sure what the "registered SQL user" fragment is meant to be/where it should go?